extensions/amp-access/0.1/jwt.js

```diff
@@ -29,6 +31,22 @@ import {tryParseJson} from '../../../src/json';
  */
 let JwtTokenInternalDef;
 
+/**
+ * Converts a text in PEM format into a binary array buffer.
+ * @param {string} pem
+ * @return {!Uint8Array}
+ * @visibleForTesting
+ */
+export function pemToBytes(pem) {
```

Comment:
This function existed in `src/utils` for 4 years and was only ever used by this extension; it can live here for now.